tests/chainerx_tests/unit_tests/routines_tests/test_manipulation.py

```diff
@@ -772,6 +772,49 @@ def forward_xp(self, inputs, xp):
         return tuple(b)
 
 
+@op_utils.op_test(['native:0', 'cuda:0'])
+@chainer.testing.parameterize_pytest('shape,indices_or_sections', [
+    ((2, 4), []),
+    ((2, 4), [2, 4]),
+    ((2, 4), [2, -3]),
+    ((2, 4), [2, 8]),
+    ((2, 4, 6), [4, 2]),
+    ((2, 4, 6), [1, 3]),
+    ((2, 4, 5), [1, -3]),
+    ((2, 4, 8), [2, 5]),
+    ((2, 4, 10), [1, 4]),
+    ((2, 6), numpy.array([1, 2])),  # indices with 1-d numpy array
+    ((2, 6), numpy.array([2])),  # indices with (1,)-shape numpy array
+    ((2, 4, 8), numpy.array(2)),  # sections numpy scalar
+    ((2, 4, 6, 8), numpy.array(2.0)),  # sections with numpy scalar, float
+    ((2, 4, 6, 8), 2.0),  # float type sections, without fraction
+    # indices with empty numpy indices
+    ((2, 4, 8, 10), numpy.array([], numpy.int32)),
+    ((2, 4, 5, 10), numpy.array([], numpy.float64)),
+])
+class TestVSplit(op_utils.NumpyOpTest):
+
+    def setup(self):
+        # TODO(niboshi): There's a bug in backward of split() in which the
```

Comment:
Although copied, I prefer having the actual author's name before the comment (i.e. niboshi is not responsible for these lines of code). Could you write your own?